slasher/db/indexed_attestations.go

```diff
@@ -117,7 +118,7 @@ func (db *Store) SaveIndexedAttestation(idxAttestation *ethpb.IndexedAttestation
 
 	// prune history to max size every 10th epoch
 	if idxAttestation.Data.Source.Epoch%10 == 0 {
```

Comment:
why 10eth epoch? would be nice to avoid magic number, declare it as a const?

Reply:
thanks! done